products/sle/main.pm

```diff
@@ -758,7 +758,12 @@ sub load_consoletests {
     }
     #have SCC repo for SLE product
     if (have_scc_repos()) {
-        loadtest "console/yast_scc";
+        if (sle_version_at_least '15') {
+            loadtest "console/proxyscc_postreg";
```

Comment:
SUSEConnect was already there in before. I don't see a need to that as an alternative. How about specifying a specific test variable for this mode?

Reply:
yast_scc is the way how it was done before, from sle15 we do it differently, isn't it?

Reply:
I don't see a reason how sle15 should behave different regarding the use of SUSEConnect except for the list of modules